README.md

```diff
@@ -52,17 +52,40 @@ swp_bridge:
 
 ### Step 4: Configure the SWPBridgeBundle
 
-Add the following parameter to your `parameters.yml` file.
+Add the following parameters to your `parameters.yml` file.
```

Comment:
why into `parameters.yml` file? the config should be placed into `config.yml` file. Look at your `Configuration.php` class which is responsible for validating and merging config files not `parameters.yml`